Gemfile.lock

```diff
@@ -182,64 +182,62 @@ GEM
       addressable (~> 2.7)
     letter_opener (1.7.0)
       launchy (~> 2.2)
-    listen (3.4.1)
+    listen (3.5.1)
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
-    loofah (2.9.0)
+    loofah (2.9.1)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
     mail (2.7.1)
       mini_mime (>= 0.1.1)
-    marcel (0.3.3)
-      mimemagic (~> 0.3.2)
+    marcel (1.0.1)
     method_source (1.0.0)
-    mimemagic (0.3.5)
-    mini_mime (1.0.2)
-    minitest (5.14.3)
+    mini_mime (1.0.3)
```

Comment:
### [mini_mime](https://github.com/discourse/mini_mime)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.0.2 → 1.0.3

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/discourse/mini_mime/compare/v1.0.2...v1.0.3).

These are the individual commits:
- (b588e0f) [List .lookup_by_extension in the README (#28)](https://github.com/discourse/mini_mime/commit/b588e0f876d0bc36c30e7581d397b813d26e69c8)
- (ecaaffd) [DEV: Switch from Travis to Actions (#29)](https://github.com/discourse/mini_mime/commit/ecaaffd63fe5cc86cdc3cbef42cde0aa81e47832)
- (7ea1a8d) [Update to the latest mime-types-data (#31)](https://github.com/discourse/mini_mime/commit/7ea1a8d41def35a7ac90fe33a0b7465bf79b823c)
- (d2d90f1) [Automate DB updates with scheduled GH Action](https://github.com/discourse/mini_mime/commit/d2d90f1c96d9fb7006ecdb8cad61af480ddfefe1)
- (5037108) [Simplify gem caching for build job](https://github.com/discourse/mini_mime/commit/5037108b4c9d9ea99dd03f28b0bb4973c4ffa018)
- (0a64fed) [Abort if steps fail in automated DB update](https://github.com/discourse/mini_mime/commit/0a64fede773f464fba8e8b8602a0623e832d3176)
- (53d2415) [Update mime-types-data before rebuilding DB](https://github.com/discourse/mini_mime/commit/53d241543ba2b82547c7332e88a912752d8f6e32)
- (57b9264) [Rebuild DBs to fix specs](https://github.com/discourse/mini_mime/commit/57b92645260547aaad2e91fe15b8359462a75712)
- (e20df78) [Version bump and database update](https://github.com/discourse/mini_mime/commit/e20df780a9024ceb71322cbacd335caf548900d9)


</details>